### DISCUSSION TITLE
✅ 🔗 Allow to set Custom Preloaders (Duplicate)

### DISCUSSION THREAD

**Discussion by jehaw** [elementor/elementor#14250] [Feature Request]
It would be greatly helpful not only for me, but for many others I am sure, if Elementor provided a Template under the Template Builder for Loading Panes. As of now you have to either download external plugins (and hope there is no conflict) to create a loading pane, or do a semi-extensive hack in the css and function.php files to show one.

What are the chances?

Thanks in advance,

J. Hawley
jeHaw Webdesign
Germany

**Comment by jehaw** [comment]
Just to clarify, I mean the preloader for when loading a page.

Jim

**Comment by 79ho3ein** [comment]
Yes, in my opinion, elementor needs a loading page to get better. This feature is really needed. Like page 404 in elementor theme builder.
Please add this feature to elementor sooner. thanks

**Comment by nicholaszein** [comment]
Hi!

We are currently in the process of organizing all Feature Request. And we have great news!

### 📢 We're happy to announce that this feature was released with Elementor Pro 3.6.0 with the introduction of Page Transitions! 🥳 

✅ Feel free to check it out and update your plugin to the new version if you haven't already!

https://elementor.com/help/page-transitions/

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
